__tests__/integration-tests.js

```diff
@@ -163,4 +163,43 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(body.comments).toHaveLength(0)
         })
     })
+})
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: responds with newly created comment object', () => {
+        return request(app)
+        .post('/api/articles/7/comments')
+        .send({username: 'rogersop', body: 'A man has got to eat.'})
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toMatchObject({author: 'rogersop', body: 'A man has got to eat.', article_id: 7})
```

Comment:
This is a good start, but we should be expecting more properties from this comment object. It seems like your example response in endpoints.json has all of the needed keys but you are not testing to ensure they are all coming through here.